docs/user-guide/device-metrics.md

```diff
@@ -50,4 +50,128 @@ spec:
     system:
       allowList: 
           name: system-allow-list
+```
+## Metrics receiver
+### Overview
+The devices can be configured to write the metrics to a remote server. The client in the device uses [Prometheus Remote Write API](https://docs.google.com/document/d/1LPhVRSFkGNSuU1fBd81ulhsCPR4hkSZyyBj1SZ8fWOM/edit#heading=h.p12mxouu8g0h) (see also [Prometheus Integrations](https://prometheus.io/docs/operating/integrations/)). The device writes metrics until all are written. It then waits 5 minutes for more metrics to be collected.
+### Configuration
+The feature is disabled by default. It is configured via EdgeDevice/EdgeDeviceGroup CRs. Example with inline documentation and defaults:
+```yaml
+spec:
+    metrics:
+      receiverConfiguration:
+        caSecretName: receiver-tls # secret containing CA cert. Secret key is 'ca.crt'. optional
```

Comment:
`Optional.` - capital letter after a full stop in this and subsequent lines.

Reply:
done